Packs/ServiceNow/ReleaseNotes/2_5_10.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ServiceNow v2
+- Added the *Mirrored XSOAR Ticket custom close state code* parameter, which allows you to define the custom Service Now close states codes that correspond to the XSOAR incident custom close reasons defined in the server configuration.
```

Comment:
```suggestion
Added the *Mirrored Cortex XSOAR Ticket custom close state code* parameter, which allows you to define the custom Service Now close states codes that correspond to the Cortex XSOAR incident custom close reasons defined in the server configuration.
```

Reply:
Not changed since it effect configuration parameter naming.